### PR TITLE
scnim

### PR DESCRIPTION
**Pull request by capocasa**
I think everything's as it should be, let me know if I missed something

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -8593,5 +8593,23 @@
     "description": "Text editor to speed up testing code snippets",
     "license": "MIT",
     "web": "https://github.com/genotrance/snip"
+  },
+  {
+    "name": "scnim",
+    "url": "https://github.com/capocasa/snip",
```
Comment (dom96) [inline packages.json]: This URL gives a 404.

**Comment by capocasa** [comment]
Weird, my branch is up to date, but my pull request is missing a commit. I'll look into it

**Comment by capocasa** [comment]
Seems to be correct now, not sure what changed

**Comment by dom96** [comment]
Still not seeing the changes. Maybe you committed to a different branch?

**Comment by capocasa** [comment]
Nope- correct branch.

If I click "view changes" on your "requested changes" message, and scroll down to the patch, I get the correct URL, ending with scnim. Maybe the incorrect one got cached as a preview here in the thread view?

**Comment by dom96** [comment]
Weird. I checked your fork repo too and didn't see the correct code so I just modified it manually.

**Comment by capocasa** [comment]
Yeah- thanks!

**Pull request merged**